glustercli/cmd/volume.go

```diff
@@ -267,8 +267,28 @@ var volumeGetCmd = &cobra.Command{
 	Short: helpVolumeGetCmd,
 	Args:  cobra.RangeArgs(1, 2),
 	Run: func(cmd *cobra.Command, args []string) {
-		volname := cmd.Flags().Args()[0]
-		fmt.Println("GET:", volname)
+		if len(cmd.Flags().Args()) == 2 {
+			volname := cmd.Flags().Args()[0]
+			optname := cmd.Flags().Args()[1]
+
+			opts, err := client.VolumeGet(volname, optname)
+			if err != nil {
+				log.WithFields(log.Fields{
+					"error": err.Error(),
+				}).Error("error getting volume options")
+				failure("Error getting Volume options", err, 1)
+			}
+
+			table := tablewriter.NewWriter(os.Stdout)
+			table.SetHeader([]string{"Name", "Value", "DefaultValue", "Modified"})
```

Comment:
"Modified" can be second column, easy readability.

Also change the display for Modified column as "Yes/No" instead of "true/false"